wodles/gcloud/tests/test_tools.py

```diff
@@ -28,28 +41,21 @@ def test_get_script_arguments(capsys):
 def test_get_script_arguments_required(capsys, args):
     """Test get_script_arguments shows an error message when the required parameters are not provided."""
     with patch("sys.argv", args), pytest.raises(SystemExit) as exception:
-        tools.get_script_arguments()
+        get_script_arguments()
     stdout, stderr = capsys.readouterr()
     assert stdout == "", 'The output was not empty'
     assert stderr != "", 'No error message was found in the output'
     assert exception.value.code == 2
 
 
-@pytest.mark.parametrize('level, expected_logging_level', [
-    (0, logging.WARNING),
-    (1, logging.INFO),
-    (2, logging.DEBUG)
+@pytest.mark.parametrize("log_level, log_method, expected_message", [
+    (logging.INFO, 'info', 'Test info message'),
+    (logging.DEBUG, 'debug', 'Test debug message'),
+    (logging.WARNING, 'warning', 'Test warning message'),
+    (logging.ERROR, 'error', 'Test error message'),
 ])
-@patch('tools.logging.getLogger')
-def test_get_stdout_logger(mock_logger, level, expected_logging_level):
-    """Test the get_stdout_logger function created a logger object with the expected logging level value."""
-    with patch('tools.logger') as mock_l:
-        mock_l.setLevel = MagicMock()
-        tools.get_stdout_logger(name='test', level=level)
-        mock_l.setLevel.assert_called_with(expected_logging_level)
-
-
-def test_arg_valid_date():
-    """Test arg_valid_dates raises an error when a date parameter doesn't have a valid format."""
-    with pytest.raises(argparse.ArgumentTypeError):
-        tools.arg_valid_date('invalid_date')
+def test_log_methods(gcp_logger, caplog, log_level, log_method, expected_message):
```

Comment:
The logging is no longer handled in the `tools.py` file unlike it was with the `get_stdout_logger` function, this test could be removed.

Reply:
Moved